src/OBD2/OBD2.c

```diff
@@ -56,6 +56,13 @@ static float kph_to_mph(float kph)
     return kph * 1.60934;
 }
 
+static float kPa_to_psig(float kPa)
```

Comment:
Please add a comment about the meaning of `psig`.  I never knew the relative term.  TIL.